src/index.html

```diff
@@ -7,8 +7,84 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo-image"
+        >
+      </a>
+
+      <nav class="navbar">
```

Comment:
Correct bem naming convention would look like so `class="nav header__nav"as you are declaring another block here, it should contain class from the previous one and also you should use the same block name as it's child elements that's why nav